### ISSUE TITLE
Provenance.json file

### ISSUE THREAD

**Issue by kmexter** [marco-bolo/dataset-catalogue#33]
Dataset descriptions will need to be supplimented with provenance.jsonld files
We have to make a template for this
Then see if we can harvest anything into it from our downloadURLs or from the tasks manually
Even if they cannot do much, it is worth doing to see what is/is not currently possible

**Comment by pbuttigieg** [comment]
@kmexter I'm not sure I follow. Most if not all PROV elements can be covered with native schema.org Types and properties 

Xref https://github.com/iodepo/odis-arch/pull/451

**Comment by kmexter** [comment]
indeed but we need to decide what our template will include so that the WPs can fill it in properly. for now this is a placeholder, but should look at this again in Jan before the GA.

**Comment by kmexter** [comment]
> @kmexter I'm not sure I follow. Most if not all PROV elements can be covered with native schema.org Types and properties
> 
> Xref [iodepo/odis-arch#451](https://github.com/iodepo/odis-arch/pull/451)

@pbuttigieg Is there then a mapping of schema to PROV? Because if so, we REALLY could use it. The link you have is a pull request which I cannot follow well enough to find the mapping file...

**Comment by pbuttigieg** [comment]
> We are going to throw together a provenance description that is suitable for our MBO use case, which is where the source data already exist and have been obtained, QCd, maybe subselected, subjected to some code, and then maybe QCd or reformatted. So this is all digital work. The EMBRC prov model is more about the source data - biological material to digital results - but we can take the digital parts of that to make an MBO-prove.json template. We are really hoping that @pbuttigieg (@roblinksdata also?) can provide the PROV<->schema mapping because we use a lot of prov in our model.

I can update the ODIS guidance (it's due anyway https://github.com/iodepo/odis-arch/issues/414) to cover these cases. Is the following the complete list?

- [ ] harvest of source data
- [ ] quality control of data
- [ ] subsetting of data
- [ ] processing / transformation of data by software
  - [ ] reformatting as a processing step 
- [ ] production of new data 


> There may be MBO tasks that also produce raw data. They _should_ then follow the full EMBRC prov model BUT (1) it is scary compared to the much smaller one we are making here and (2) we have a few difficult issues to still figure out (in particular for processes that start with material and end with data - is that a material activity or a digital activity?), so I am not sure will be 100% done by the GA.

The data should not use an EMBRC model - that's siloing if it isn't entirely compatible with a more generic approach. EMBRC is not a data standard issuer. The concepts of material vs digital activity show semantic confounding already, for example. Give us a shot to produce some global guidance.

**Comment by kmexter** [comment]
yes, those steps are what we were thinking would be required. Hope to have something to post in the new templates folder by next week

**Comment by kmexter** [comment]
@laurianvm can you add the entire EMBRC prov model to https://github.com/marco-bolo/metadata-catalogue/tree/main/templates/provenance? The one there at present - dataset-provenance.jsonld is for the digital-to-digital steps only, but actually we need the material steps also.
Perhaps add the new bits to the existing file rather than adding a new one. Bear in mind that you made some changes to the existing file after you put it there, probably best to not overwrite those.
Then we can close this issue and https://github.com/marco-bolo/metadata-catalogue/issues/49 and raise a new one to take it from there.
